### ISSUE TITLE
flake8 not run against all sources from tox

### ISSUE THREAD

**Issue by yazug** [release-depot/koji_wrapper#21]
* koji_wrapper version: 0.1.0

### Description

tox flake8/pep8/linting check doesn't actually checking flake8 linting results for all source files

### What I Did

```
tox

... no issues

$ pipenv run flake8 tests/ koji_wrapper/ docs/ setup.py
tests/conftest.py:5:1: E302 expected 2 blank lines, found 1
tests/conftest.py:32:1: E302 expected 2 blank lines, found 1
tests/conftest.py:81:60: W291 trailing whitespace
tests/conftest.py:163:1: W391 blank line at end of file
tests/unit/test_koji_wrapper.py:13:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:17:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:18:24: E201 whitespace after '('
tests/unit/test_koji_wrapper.py:18:28: E251 unexpected spaces around keyword / parameter equals
tests/unit/test_koji_wrapper.py:18:30: E251 unexpected spaces around keyword / parameter equals
tests/unit/test_koji_wrapper.py:19:31: E251 unexpected spaces around keyword / parameter equals
tests/unit/test_koji_wrapper.py:19:33: E251 unexpected spaces around keyword / parameter equals
tests/unit/test_koji_wrapper.py:20:32: E251 unexpected spaces around keyword / parameter equals
tests/unit/test_koji_wrapper.py:20:34: E251 unexpected spaces around keyword / parameter equals
tests/unit/test_koji_wrapper.py:22:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:32:43: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:34:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:42:43: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:44:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:52:43: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:55:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:64:24: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:67:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:76:27: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:80:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:86:80: E501 line too long (81 > 79 characters)
tests/unit/test_koji_wrapper.py:89:26: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:91:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:99:67: E231 missing whitespace after ':'
tests/unit/test_koji_wrapper.py:103:25: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:106:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:120:31: E231 missing whitespace after ','
tests/unit/test_koji_wrapper.py:122:1: E302 expected 2 blank lines, found 1
tests/unit/test_koji_wrapper.py:122:80: E501 line too long (82 > 79 characters)
tests/unit/test_koji_wrapper.py:134:9: F841 local variable 'srpm_url' is assigned to but never used
tests/unit/test_koji_wrapper.py:135:1: W391 blank line at end of file
koji_wrapper/base.py:10:1: F401 'operator' imported but unused
koji_wrapper/base.py:12:1: E302 expected 2 blank lines, found 1
koji_wrapper/base.py:41:80: E501 line too long (80 > 79 characters)
koji_wrapper/base.py:58:80: E501 line too long (80 > 79 characters)
koji_wrapper/base.py:77:80: E501 line too long (80 > 79 characters)
koji_wrapper/wrapper.py:11:1: E302 expected 2 blank lines, found 1
koji_wrapper/wrapper.py:35:9: E265 block comment should start with '# '
koji_wrapper/wrapper.py:61:80: E501 line too long (80 > 79 characters)
koji_wrapper/wrapper.py:67:37: F821 undefined name 'self'
docs/conf.py:25:1: E402 module level import not at top of file
docs/conf.py:27:80: E501 line too long (80 > 79 characters)
docs/conf.py:30:1: E402 module level import not at top of file
docs/conf.py:32:1: E302 expected 2 blank lines, found 1
docs/conf.py:37:1: E305 expected 2 blank lines after class or function definition, found 1
docs/conf.py:42:1: E303 too many blank lines (3)
docs/conf.py:178:1: W391 blank line at end of file

```

**Comment by jguiditta** [comment]
As you noted in my pr #18, I fixed these as well, but I agree they make more sense to include as a separate commit/PR

**Comment by yazug** [comment]
closed with Pull Request #22
